qtism/runtime/rendering/markup/xhtml/XhtmlRenderingEngine.php

```diff
@@ -107,53 +113,59 @@ public function __construct()
-        $this->registerRenderer('gapMatchInteraction', new GapMatchInteractionRenderer());
-        $this->registerRenderer('inlineChoiceInteraction', new InlineChoiceInteractionRenderer());
-        $this->registerRenderer('textEntryInteraction', new TextEntryInteractionRenderer());
-        $this->registerRenderer('extendedTextInteraction', new ExtendedTextInteractionRenderer());
-        $this->registerRenderer('hottextInteraction', new HottextInteractionRenderer());
-        $this->registerRenderer('hotspotInteraction', new HotspotInteractionRenderer());
         $this->registerRenderer('selectPointInteraction', new SelectPointInteractionRenderer());
-        $this->registerRenderer('graphicOrderInteraction', new GraphicOrderInteractionRenderer());
-        $this->registerRenderer('graphicGapMatchInteraction', new GraphicGapMatchInteractionRenderer());
-        $this->registerRenderer('positionObjectInteraction', new PositionObjectInteractionRenderer());
-        $this->registerRenderer('sliderInteraction', new SliderInteractionRenderer());
-        $this->registerRenderer('mediaInteraction', new MediaInteractionRenderer());
-        $this->registerRenderer('drawingInteraction', new DrawingInteractionRenderer());
-        $this->registerRenderer('uploadInteraction', new UploadInteractionRenderer());
-        $this->registerRenderer('endAttemptInteraction', new EndAttemptInteractionRenderer());
-        $this->registerRenderer('simpleChoice', new SimpleChoiceRenderer());
         $this->registerRenderer('simpleAssociableChoice', new SimpleAssociableChoiceRenderer());
-        $this->registerRenderer('inlineChoice', new InlineChoiceRenderer());
+        $this->registerRenderer('simpleChoice', new SimpleChoiceRenderer());
         $this->registerRenderer('simpleMatchSet', new SimpleMatchSetRenderer());
-        $this->registerRenderer('hottext', new HottextRenderer());
-        $this->registerRenderer('gapText', new GapTextRenderer());
-        $this->registerRenderer('gapImg', new GapImgRenderer());
-        $this->registerRenderer('gap', new GapRenderer());
-        $this->registerRenderer('hotspotChoice', new HotspotChoiceRenderer());
-        $this->registerRenderer('associableHotspot', new AssociableHotspotRenderer());
-        $this->registerRenderer('positionObjectStage', new PositionObjectStageRenderer());
-        $this->registerRenderer('assessmentItem', new AssessmentItemRenderer());
-        $this->registerRenderer('printedVariable', new PrintedVariableRenderer());
+        $this->registerRenderer('sliderInteraction', new SliderInteractionRenderer());
+        $this->registerRenderer('stylesheet', new StylesheetRenderer());
+        $this->registerRenderer('table', new TableRenderer());
+        $this->registerRenderer('textEntryInteraction', new TextEntryInteractionRenderer());
+        $this->registerRenderer('textRun', new TextRunRenderer());
+        $this->registerRenderer('uploadInteraction', new UploadInteractionRenderer());
```

Comment:
i hope you sure that nothing is missing :)

Reply:
I try to do them more accurate

Reply:
all revertend and make changes more understandable